.pre-commit-config.yaml

```diff
@@ -201,6 +201,7 @@ repos:
     - lxml  # dep of `--txt-report`, `--cobertura-xml-report` & `--html-report`
     - msrestazure  # needed by credentials.azure_kv
     - pytest
+    - pytest_mock
```

Comment:
This will disappear post-rebase. The PR is blocked on conflict resolution: https://results.pre-commit.ci/run/github/836873755/1729693574.gKfyS5-rR2mqX3tQ-7MlzA.

Reply:
marking as resolved